src/GeekLearning.Templating.Handlebars/HandlebarsTemplateProvider.cs

```diff
@@ -33,6 +34,44 @@ private class Scope : ITemplateProviderScope
             public Scope()
             {
                 this.handlebars = Handlebars.Create();
+
+                this.handlebars.RegisterHelper("format", (writer, context, arguments) =>
+                {
+                    if (arguments.Length <= 1)
+                    {
+                        return;
+                    }
+
+                    var format = "";
+                    if (arguments.Length > 1)
+                    {
+                        format = arguments[1] as string ?? "";
+                    }
+
+                    var culture = System.Globalization.CultureInfo.InvariantCulture;
```

Comment:
I think the default needs to be the current Context Culture. In that way, if someone is using it in a web context and uses the builtin culture support, he won't have to explicitly set the culture. 

In addition InvariantCulture is more a serialization culture unlikely to be used in a template.